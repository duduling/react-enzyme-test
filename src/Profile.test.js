import React from 'react'
import { render, getByText } from '@testing-library/react'

import Profile from './Profile'

describe('<Profile />', () => {
    it('matches snapshot', () => {
        const { container } = render(<Profile username="Duduling" name="두들링" />)
        expect(container).toMatchSnapshot()
    })
    it('shows ther props correctly', () => {
        const { container } = render(<Profile usename={'Duduling'} name={'두들링'} />)

        getByText(container, 'Duduling')
        getByText(container, '(두들링)')
        getByText(container, /김/)
    })
})