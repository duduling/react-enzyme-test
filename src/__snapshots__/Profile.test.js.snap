// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matches snapshot 1`] = `
<div>
  <div>
    <b>
      Duduling
    </b>
    <span>
      (
      두들링
      )
    </span>
  </div>
</div>
`;
